lib/serializables/log-event.js

```diff
@@ -51,9 +51,6 @@ function LogEventFactory (
             host: {
                 type: 'string'
             },
-            module: {
```

Comment:
The module name is duplicated with the file name, file name could give us the same information, and the module name is the same with file name it's just for developer, not for normal users. or another way is add module name to file name line, although it's duplicated, it could be quick map for module and file